lms/djangoapps/course_blocks/transformers/tests/test_start_date.py

```diff
@@ -0,0 +1,75 @@
+"""
+Tests for StartDateTransformer.
+"""
+import ddt
+from datetime import timedelta
+from django.utils.timezone import now
+from mock import patch
+
+from ..start_date import StartDateTransformer, DEFAULT_START_DATE
+from .test_helpers import BlockParentsMapTestCase, update_block
+
+
+@ddt.ddt
+class StartDateTransformerTestCase(BlockParentsMapTestCase):
```

Comment:
Yes. Good catch.  That's exactly the unit test I was referring to earlier that we're currently missing, which I am going to add while you guys review.  (There are 3 other unit tests I also have in mind.  Can you catch them?  ha ha.)